app/styles/_story.scss

```diff
@@ -30,6 +30,22 @@ article > section a:hover {
   }
 }
 
+@mixin with-gradient($gradient) {
```

Comment:
this is defined in `nypr-ui/_hero.scss`, which is imported in this app's `app.scss`, so you should be able to invoke it without redefining it here.